cli/send.go

```diff
@@ -128,7 +147,7 @@ var SendCmd = &cli.Command{
 			params.Params = decparams
 		}
 
-		if ethtypes.IsEthAddress(params.From) {
+		if ethtypes.IsEthAddress(params.From) || ethtypes.IsEthAddress(params.To) {
```

Comment:
I think this is going to cause some issues.

1. There are almost certainly users using native accounts to send messages to EVM smart contracts, explicitly specifying the method number.
2. Implicitly calling `InvokeContract` is the _nice_ thing to do, but that's a significant change of behavior.

Ideally this would have this method number handling if and only if the user passes a literal `0x...` address in the to field. If they pass a normal Filecoin address, they should get the normal Filecoin method handling.

Reply:
pulled this up into a new issue https://github.com/filecoin-project/lotus/issues/12326